src/components/features/directory/DirectoryCard/DirectoryCard.tsx

```diff
@@ -180,9 +180,10 @@ export function DirectoryCard({
                                     surname,
                                 })}
                             </ProviderName>
-                            {licenses && (
+                            {credentials && (
                                 <ProviderCredentials>
-                                    {licenses}
+                                    {/* // TODO: What should this be? */}
+                                    {JSON.stringify(credentials)}
```

Comment:
This was a build issue after `licenses` was removed off of the provider profile schema. Not sure what this is supposed to look like in UI, but it needs to be updated